### ISSUE TITLE
Deploy does not work with angular 13

### ISSUE THREAD

**Issue by tiberiuzuld** [angular-schule/angular-cli-ghpages#137]
Current latest version does not work with angular v13
```
> angular-gridster2@13.0.0 deploy
> ng deploy

An unhandled exception occurred: Cannot find module '@angular-devkit/architect'
Require stack:
- /home/runner/work/angular-gridster2/angular-gridster2/node_modules/angular-cli-ghpages/deploy/builder.js
- /home/runner/work/angular-gridster2/angular-gridster2/node_modules/@angular/cli/node_modules/@angular-devkit/architect/node/node-modules-architect-host.js
- /home/runner/work/angular-gridster2/angular-gridster2/node_modules/@angular/cli/node_modules/@angular-devkit/architect/node/index.js
- /home/runner/work/angular-gridster2/angular-gridster2/node_modules/@angular/cli/models/architect-command.js
- /home/runner/work/angular-gridster2/angular-gridster2/node_modules/@angular/cli/commands/deploy-impl.js
- /home/runner/work/angular-gridster2/angular-gridster2/node_modules/@angular/cli/node_modules/@angular-devkit/schematics/tools/export-ref.js
- /home/runner/work/angular-gridster2/angular-gridster2/node_modules/@angular/cli/node_modules/@angular-devkit/schematics/tools/index.js
- /home/runner/work/angular-gridster2/angular-gridster2/node_modules/@angular/cli/utilities/json-schema.js
- /home/runner/work/angular-gridster2/angular-gridster2/node_modules/@angular/cli/models/command-runner.js
- /home/runner/work/angular-gridster2/angular-gridster2/node_modules/@angular/cli/lib/cli/index.js
- /home/runner/work/angular-gridster2/angular-gridster2/node_modules/@angular/cli/lib/init.js
See "/tmp/ng-ea1azD/angular-errors.log" for further details.
```

**Comment by JohannesHoppe** [comment]
Ok, I will work on this next week. Please use the standalone version until the issue is resolved:

https://github.com/angular-schule/angular-cli-ghpages/blob/master/docs/README_standalone.md

**Comment by euaaron** [comment]
Just adding more info, this is what happens during the installation of `angular-cli-ghpages` under an Angular 13 project generated by `@angular/cli`:

```js
yarn ng add angular-cli-ghpages
yarn run v1.22.17
$ ng add angular-cli-ghpages
ℹ Using package manager: yarn✔ Found compatible package version: angular-cli-ghpages@0.6.2.✔ Package information loaded.

The package angular-cli-ghpages@0.6.2 will be installed and executed.Would you like to proceed? Yes
✔ Package successfully installed.An unhandled exception occurred: NOT SUPPORTED: keyword "id", use "$id" for schema ID  
See "C:\Users\euaar\AppData\Local\Temp\ng-WkWfhC\angular-errors.log" for further details.
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by JohannesHoppe** [comment]
I just published `angular-cli-ghpages@1.0.0-rc.3` to npm! This version supports Angular 13.
Many thanks to @fmalcher for the patch!

**Comment by JohannesHoppe** [comment]
Hello @euaaron, please take a look at the version resolution of yarn. The latest version is v1.0.0-rc.3.
The version in the error message is0.6.2, which is very old.

To force the right version, you can use:

```
ng add angular-cli-ghpages@1.0.0-rc.3
```

**Comment by euaaron** [comment]
> Hello @euaaron, please take a look at the version resolution of yarn. The latest version is v1.0.0-rc.3.
> The version in the error message is 0.6.2, which is very old.
> 
> To force the right version, you can use:
> 
> ```
> ng add angular-cli-ghpages@1.0.0-rc.3
> ```

Oh, I forgot to say that I've solved by installing with `yarn add -D angular-cli-ghpages@next` instead of using `ng add`.

But i will try your tip too. Thanks!

**Comment by fmalcher** [comment]
@euaaron Iin the meantime, we released 1.0.0 so you should be able to do `ng add` again as usual. 🙂 Let us know if there are any issues!
